chart/README.md

```diff
@@ -45,3 +45,54 @@ Parameter | Description | Default
 
 All other user-configurable settings, default values and some commentary about them can be found in [values.yaml](values.yaml).
 
+# FusionAuth
+
+[![Build Status](https://travis-ci.org/ninjaneers-team/fusionauth.svg?branch=master)](https://travis-ci.org/ninjaneers-team/fusionauth)
+
+[FusionAuth](https://fusionauth.io/) is a modern platform for Customer Identity and Access Management (CIAM). FusionAuth provides APIs and a responsive web user interface to support login, registration, localized email, multi-factor authentication, reporting and much more.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add ninjaneers https://ninjaneers-team.github.io/charts
```

Comment:
We probably don't want references to ninjaneers in here, and also this readme should just be in the root of the repo, added to the existing readme.